### PR TITLE
Create desktop version

### PR DESCRIPTION
**Pull request by Bifabig**
I created a responsive desktop version for the portfolio project.

- Used @media query to structure the desktop version at breakpoint 768px.
- Reimplemented the navigation bar.
- Split the `header` section into left and right.
- Restructured the projects section using CSS grid.
- Rearranged the about, contact and footer `sections` as well.

### CODE REVIEW

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @Bifabig, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

## Highlights
Used GitHub flow ✅
used linters ✅
PR summary ✅
UI ✅


## Required Changes ♻️
- [x] Kindly pay attention to the details of your design. Your button size doesn't match that of the Figma template and it's also missing the `-->` icon. See the screenshot below for more details.
Figma:
Screenshot 2023-03-31 at 09 15 25

Your design:
Screenshot 2023-03-31 at 09 14 36

- [x] Please make sure that the about and skill section are positioned side by side above `768px` screen size and wider. Currently, it's only positioned side by side above `1254px` screen size and wider. 
- [x] Also, please make sure to reduce the section size so it matches that of the Figma template. Currently, it's filled on the screen.

Figma:
Screenshot 2023-03-31 at 09 32 54

Your design:
Screenshot 2023-03-31 at 09 32 18

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -337,3 +432,410 @@ input:focus {
...
+    width: 80%;
+    height: 80%;
+    align-self: flex-end;
+    justify-self: end;
+    margin: 0 -60px -60px 0;
+  }
+
+  .get-started {
+    margin-top: 100px;
+  }
```
Comment (AbelG101) [inline style.css]: - [x] [OPTIONAL] Hey Biftu, Great job making the changes requested previously. It seems like the previous reviewer forgot to mention this but kindly add a `#f9f9fa` background color to the "Get started" section as specified in the [Figma design. 🎯](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23-9&t=t45uZqimwMP5BSMR-0)

File: index.html [hunk 2]
```diff
@@ -342,6 +422,11 @@ <h2 id="tagline">Get Started</h2>
         </div>
         <input type="submit" value="Start collaboration" class="primary-btn" />
       </form>
+      <div class="image-geometry_7">
+        <a href="#"
+          ><img src="./assets/img/image-geometry_7.png" alt="logo_desktop"
+        /></a>
+      </div>
```
Comment (AbelG101) [inline index.html:427]: - [x] [OPTIONAL] Once again I'd like to apologize as the previous reviewer forgot to mention this but it seems like you forgot to add the bottom image in the "Get started" section as specified in the [Figma design 🎨](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23-9&t=t45uZqimwMP5BSMR-0)
    - Figma design:
      firefox_SaWm8KGpLf
    - Your design:
      Inkedfirefox_e0mhYLCi7L

**Review by AbelG101 — APPROVED** [review]
# Hi @Bifabig 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- UI looking on point ✔
-  The website works like a charm ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Bifabig) [reply index.html:427]: Thank You @AbelG101

**Pull request merged**